templates/head/google_fonts.twig

```diff
@@ -4,42 +4,42 @@
 	<style>
 		@font-face {
 			font-family: Sanctuary;
-			src: url("{{ theme_uri }}/public/fonts/SanctuaryLC-Regular.otf");
+			src: url("{{ theme_uri }}/fonts/SanctuaryLC-Regular.otf");
 			font-display: swap;
 		}
 
 		@font-face {
 			font-family: "Save the Arctic";
-			src: url("{{ theme_uri }}/public/fonts/save-the-arctic.otf");
+			src: url("{{ theme_uri }}/fonts/save-the-arctic.otf");
```

Comment:
Putting this comment here because Github doesn't allow me to put it on a renamed file. It was previously inside an `arctic` subfolder, but I couldn't find anything using it from there. In the plugin it was also not in a subfolder.